core/src/syscall/precompiles/blake2b/compress/columns.rs

```diff
@@ -0,0 +1,52 @@
+use core::borrow::{Borrow, BorrowMut};
+use std::mem::size_of;
+
+use crate::memory::{MemoryReadCols, MemoryReadWriteCols};
+use sp1_derive::AlignedBorrow;
+
+use super::{
+    mix::MixOperation, NUM_MIX_ROUNDS, NUM_MSG_WORDS_PER_CALL, NUM_STATE_WORDS_PER_CALL,
+    OPERATION_COUNT,
+};
+use super::{MSG_ELE_PER_CALL, STATE_ELE_PER_CALL};
+
+pub const NUM_BLAKE2B_COMPRESS_INNER_COLS: usize = size_of::<Blake2bCompressInnerCols<u8>>();
+
+#[derive(AlignedBorrow, Default, Debug, Clone, Copy)]
+#[repr(C)]
+pub struct Blake2bCompressInnerCols<T> {
```

Comment:
how many cols does this end up being? rust analyzer should be able to tell you

Reply:
It has `351` columns.